src/frontend/Sider/Personoversikt/Behandlingsoversikt/OpprettNyBehandling/OpprettNyBehandlingModal.tsx

```diff
@@ -37,17 +31,17 @@ const OpprettNyBehandlingModal: FC<Props> = ({
     const [opprettNyBehandlingType, settOpprettNyBehandlingType] =
         useState<OpprettNyBehandlingType>();
 
-    const lukkModal = () => {
+    function lukkModal() {
         settVisModal(false);
         settOpprettNyBehandlingType(undefined);
-    };
+    }
 
     if (!erSaksbehandler) {
         return null;
     }
 
     return (
-        <StyledContainer className="py-16">
+        <div className={styles.container}>
             <Button size={'small'} variant={'secondary'} onClick={() => settVisModal(true)}>
```

Comment:
Kanskje verdt å merke seg at med CSS i stedet for SC så mister litt intensjonen med tagen når man ikke lenger kan navngi den som man vil. Veldig lett å se her at man la til div-en som en container når den heter `StyledContainer`